src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -557,6 +557,10 @@ public function removeParameter(string $parameterName) : void
         $lowerName = strtolower($parameterName);
 
         foreach ($this->node->params as $key => $paramNode) {
+            if ($paramNode->var instanceof Node\Expr\Error) {
```

Comment:
Not a fan of doing this, but it's the only way I could force `psalm` to not think `$paramNode->var` is ever a `Node\Expr\Error`. I tried using `/** @var Node\Expr\Error $paramNode->var */` but it still complained that sometimes `name` property doesn't exist (if `var` is an `Error` node)... is this the right solution do you think @muglug ?